docs/pages/Introduction/what-is-hyperbridge.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: What is Hyperbridge?
+description: Hyperbridge Protocol Specification
+---
+
+# What is Hyperbridge?
+
+Hyperbridge is the frontliner of consensus based bridges, using consensus proofs, state transition validity and consensus fault proofs to ensure secure and the integrity of transactions across chains. It is an interoperability coprocessor capable of scaling cryptographically secure, consensus & state proof-based interoperability to all blockchains.
+
+## The challenge of interoperability.
+
+Ensuring secure cross-chain communication involves verifying consensus, consensus faults, state proofs, and state transitions. However, conducting these verification processes on-chain is often prohibitively costly. Consequently, attestation networks have emerged, employing multi-sig committees to verify the state of counterpart chains. Unfortunately, these architectures have led to a collective loss of $2 billion in crypto assets (sources [1](), [2]()). We advocate for the adoption of cross-chain bridges with heightened security assumptions to pave the way for the next generation of secure cross-chain applications. This necessitates the development of a new kind of interoperability coprocessor that can scale consensus proofs, state proofs, and state transitions across multiple blockchains.
+
+{ /* The solution */}
+
+## How does it work?
+ A single state machine can quickly become bottlenecked if it is to verify consensus proofs, state proofs & (re-execute) state transitions of it’s multiple connected state machines. This necessitates that the coprocessor must be multi-threaded and capable of verifying multiple state machines in parallel.
+
+### The architecture
+Hyperbridge can verify multiple state machines, a single Polkadot consensus proof attests to the validity of all its state machines. Which allows developers to aggregate the cross-chain messages from multiple connected blockchains, which are verified on multiple parachain cores, into a single zk consensus & state proof that can be verified cheaply on any chain.
+
+### The Coprocessor model
+Since verification is too expensive to be performed onchain, this necessitates the coprocessor model. Where the computation is performed off-chain and the results of the execution are presented onchain alongside cryptographic proofs of its correct execution. The following diagram gives a visual overview of hyperbridge cores. 
```

Comment:
Hyperbridge does the consensus  and state proof verifications of all it's connected networks on-chain, because it has access to more compute.